vendor/bat-native-ledger/BUILD.gn

```diff
@@ -99,9 +99,14 @@ source_set("ledger") {
     "src/bignum.h",
     "src/ledger_impl.cc",
     "src/ledger_impl.h",
+    "src/transaction_info.cc",
+    "include/bat/ledger/transaction_info.h",
```

Comment:
this should be inside `headers` section

Reply:
> linter is failing

@ryanml linter failing for .ts/tsx files, see https://travis-ci.org/brave/brave-core/jobs/497732491

Reply:
👍 done